curriculum/Unit6/Lesson-607.md

```diff
@@ -112,8 +112,8 @@ In this project your will interview an end-user in order to determine their inte
 
 1. Select an end-user (this can be a classmate, friend, teacher, relative, etc).
 2. Create a series of questions that will allow you to understand their interests. Have the user respond to the questions and record their responses.
-3. With the end-user, design a common superclass or interface that will store information related to their interests (ie: if the end-user is interested in sports then the superclass could include information common to any player regardless of sport, such as name, number, and salary).
-4. With the end-user, design a subclasses for specific elements, characters or players that fall under the superclass structure (ie: players of your favorite sports, such as basketball, soccer or tennis could have specific attributes under the athlete superclass). Be sure to place common behavior in superclasses whenever possible, and use abstract classes as appropriate. Place sport-specific information and behavior (such as kicking, vertical jump height, or speed) into subclasses whenever possible.
+3. With the end-user, design a common superclass that will store information related to their interests (ie: if the end-user is interested in sports then the superclass could include information common to any player regardless of sport, such as name, number, and salary).
```

Comment:
- “ie” should be “_i.e._”, or shift from Latin to English with “that is,”.
- “… is interested in sports then the superclass …” needs a comma before “then”.